packages/manager/apps/dedicated/client/app/account/user/support-level/support-level.controller.js

```diff
@@ -40,8 +59,26 @@ export default class UserAccountSupportLevelCtrl {
   }
 
   areAllLevelsAvailable() {
-    return FULL_SUPPORT_LEVEL_SUBSIDIARIES.includes(
-      this.currentUser.ovhSubsidiary,
+    return !some(
+      this.supportLevels,
+      (level) => !level.isAvailable(this.currentUser.ovhSubsidiary),
     );
   }
+
+  isLevelActive(supportLevel) {
```

Comment:
move this to SupportLevel class?

Reply:
This does not only depend on `SupportLevel` but also on a `PartnerLevel` so if it should be moved to a class, I'm not sure `SupportLevel` is the one to go with